### PR TITLE
Update SuperChic to version 5.4

### PR DESCRIPTION
**Pull request by stahlleiton**
This PR updates the SuperChic generator to version 5.4 as released recently in: https://github.com/LucianHL/SuperChic/releases/tag/v5.4

The new SuperChic v5.4 source package should be copied from:  https://anstahll.web.cern.ch/anstahll/superchic/SuperChic-5.4.tar.gz   to the generator directory:  https://cms-project-generators.web.cern.ch/cms-project-generators/superchic/

@bbilin

### CODE REVIEW

File: bin/Superchic/gridpack_generation.sh [hunk 1]
```diff
@@ -24,19 +24,19 @@ create_setup(){
...
 
 install_superchic(){
-    SUPERCHIC=SuperChic-5.3
+    SUPERCHIC=SuperChic-5.4
     cd ${WORKDIR}
 
     echo "Downloading "${SUPERCHIC}
     SUPERCHICDIR=${WORKDIR}/superchic_v${SUPERCHIC//[!0-9]/}
-    wget --no-verbose --no-check-certificate https://cms-project-generators.web.cern.ch/cms-project-generators/superchic/${SUPERCHIC}.tar.gz
+    wget --no-verbose --no-check-certificate https://anstahll.web.cern.ch/anstahll/superchic/${SUPERCHIC}.tar.gz
```
[the remark this answers is not in the document]
Comment (stahlleiton) [reply bin/Superchic/gridpack_generation.sh:32]: Thanks @bbilin , but I don't see the file in : https://cms-project-generators.web.cern.ch/cms-project-generators/superchic/

**Comment by bbilin** [comment]
ah yes, by mistake i put it on one directory up. i will move.

Envoyé de mon iPhone

Le 25 nov. 2024 à 14:58, Andre Stahl ***@***.***> a écrit :

﻿

@stahlleiton commented on this pull request.

________________________________

In bin/Superchic/gridpack_generation.sh<https://github.com/cms-sw/genproductions/pull/3806#discussion_r1856662891>:

>      cd ${WORKDIR}

     echo "Downloading "${SUPERCHIC}
     SUPERCHICDIR=${WORKDIR}/superchic_v${SUPERCHIC//[!0-9]/}
-    wget --no-verbose --no-check-certificate https://cms-project-generators.web.cern.ch/cms-project-generators/superchic/${SUPERCHIC}.tar.gz
+    wget --no-verbose --no-check-certificate https://anstahll.web.cern.ch/anstahll/superchic/${SUPERCHIC}.tar.gz


Thanks @bbilin<https://github.com/bbilin> , but I don't see the file in : https://cms-project-generators.web.cern.ch/cms-project-generators/superchic/

File: bin/Superchic/gridpack_generation.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbilin) [reply bin/Superchic/gridpack_generation.sh:32]: ah yes, by mistake i put it on one directory up. i will move.

[the remark this answers is not in the document]
Comment (bbilin) [reply bin/Superchic/gridpack_generation.sh:32]: fixed the folder.

[the remark this answers is not in the document]
Comment (stahlleiton) [reply bin/Superchic/gridpack_generation.sh]: @bbilin : done, the PR is now updated reverting back the website link

**Comment by stahlleiton** [comment]
@bbilin : do you have further comments on this PR or is it ready to be merged?

**Pull request merged**